envi/archs/arm/disasm.py

```diff
@@ -1906,6 +1909,19 @@ def involvesPC(self):
     def isDeref(self):
         return True
 
+    def setOperValue(self, op, emu=None, val=None):
+        # can't survive without an emulator
+        if emu == None:
+            return None
+
+        pubwl = self.pubwl >> 2
+        b = pubwl & 1
+
+        addr = self.getOperAddr(op, emu)
+
+        fmt = ("<L", "B")[b]
```

Comment:
Gotta watch out for those "L" format chars... there are older python builds where L will be 32 bits on an x86 system and 64 bits on amd64 etc. ( due to thrashing around with gcc versions and "unsigned long" behavior.

If you meant it to always be 32 bit, plz use "I"